plugins/org.jkiss.dbeaver.ui.editors.base/src/org/jkiss/dbeaver/ui/editors/internal/EditorsResources_de.properties

```diff
@@ -44,9 +44,9 @@ dialog_morph_delimited_target_group_delim_result = Ergebnis Trennzeichen
 dialog_morph_delimited_target_group_leading_text = F\u00FChrender Text
 dialog_morph_delimited_target_group_trailing_text = Nachlaufender Text
 
-database_editor_command_save_name = Speichern ...
-database_editor_command_save_tip = Vorschau\u00E4nderungen und speichern
+database_editor_command_save_name = Speichern
+database_editor_command_save_tip = Alle\u00C4nderungen speichern
 database_editor_command_revert_name = Zur\u00FCcksetzen
-database_editor_command_revert_tip = Alle \u00C4nderungen im aktuellen Editor zur\u00FCcksetzen
+database_editor_command_revert_tip = Alle \u00C4nderungen zur\u00FCcksetzen
```

Comment:
Not sure that we need to shorten tips like:
`Reset all changes in the current editor` -> `Reset all changes`
`Preview changes and save` -> `Save all changes`

Reply:
Why are the missleading translations better than the new ones?

Reply:
Could you explain, please, why the previous translation is misleading? Maybe we could come up with something better.

Reply:
okay - let's check line by line:

**database_editor_command_save_name** --> I'm totally fine with "Speichern ..." although it is not really common to use it in this way
**database_editor_command_save_tip** --> "Vorschauänderungen und speichern" is not German speech. Better is "Alle\u00C4nderungen speichern" which is "save all changes" which definitely better describes what pgm will do
**database_editor_command_revert_tip** --> why we need to state that we're talking about changes in current editor. But if really needed we could leave it as "Alle Änderungen im aktuellen Editor zurücksetzen"

what do you think?

Reply:
I think it's important to state that changes will be reset only for current editor in `database_editor_command_revert_tip`.
About `database_editor_command_save_tip`: I'm not sure how to express in German thought about not only save, but also preview changes. If there is no good way for that, then let's change it as you propose. 

Thank you so much for your work and explanation.